### PR TITLE
[UNIT TEST] New Query

### PR DESCRIPTION
**Pull request by riley-stride**
## Summary
- Add simple unit test for interchainquery `NewQuery` (tests `GenerateQueryHash` too)
- Add `keeper_test.go` for `x/interchainquery` module

## Testing
make test-unit

### CODE REVIEW

File: x/interchainquery/keeper/new_query_test.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+		request:      request,
+		period:       period,
+		callbackId:   callbackId,
+		ttl:          ttl,
+		height:       height,
+	}
+}
+
+func (s *KeeperTestSuite) TestNewQuerySuccessful() {
+	ctx := s.Ctx
```
Comment (asalzmann) [inline x/interchainquery/keeper/new_query_test.go:51]: Can you move this variable initialization into `SetupNewQuery`?

**Review by asalzmann — COMMENTED** [review]
one minor comment, otherwise lgtm!

File: x/interchainquery/keeper/new_query_test.go (hunk 1 above)
Comment (riley-stride) [reply x/interchainquery/keeper/new_query_test.go:51]: yep good idea

Comment (riley-stride) [reply x/interchainquery/keeper/new_query_test.go:51]: done

**Review by asalzmann — APPROVED** [review]

**Pull request merged**